pages/tidal tools/machine_stats.md

```diff
@@ -166,7 +166,7 @@ For piping the results straight to your Tidal workspace with Tidal Tools, you ca
 | tidal sync servers
 ```
 
-If you plan on using this approach, remember to log into `tidal tools` with `tidal login`.
+If you plan on using this approach, remember to save your password locally using `tidal config set tidal.password 'your-password'`.
```

Comment:
```suggestion
If you plan on using this approach, remember to save your password locally using `tidal config set tidal.password 'your-password'`. This will allow Tidal Tools to authenticate with the API indefinitely, as long as the configuration remains set. Note, the password is encrypted at rest using AES-256 encryption.
```